releasenotes/releasenotes_V0.6.1.md

```diff
@@ -0,0 +1,100 @@
+
+</p>
+</details>
+
+
+<details><summary>Keptn Core</summary>
+<p>
+
+- *jmeter-service:*
+  - [KEP 0005](https://github.com/keptn/enhancement-proposals/blob/master/text/0005-deployment-finished-with-deployed-endpointurl.md) - Use deploymentURILocal/deploymentURIPublic instead of guessing the service URL [#1403](https://github.com/keptn/keptn/issues/1403)
+- *helm-service:* 
+  - Create namespaces on demand and not by default [#1417](https://github.com/keptn/keptn/issues/1417)
+  - [KEP 0005](https://github.com/keptn/enhancement-proposals/blob/master/text/0005-deployment-finished-with-deployed-endpointurl.md) - Send deploymentURIPublic and deploymentURILocal after successful deployment [#1417](https://github.com/keptn/keptn/issues/1417)
+  - Only wait for deployments contained in Helm release [#1225](https://github.com/keptn/keptn/issues/1225)
+- *configuration-service:* 
+  - Allow reading files without using git pull [#1396](https://github.com/keptn/keptn/issues/1396)
+  - Improve mutex per project [#1395](https://github.com/keptn/keptn/issues/1395)
+  - Enhance API call GET `/projects` to return more information [#1394](https://github.com/keptn/keptn/issues/1394)
+- API: Allow to set `keptnContext` in events sent to API endpoint [#1355](https://github.com/keptn/keptn/issues/1355)
+
+</p>
+</details>
+
+<details><summary>CLI Enhancements</summary>
+<p>
+
+- Check if a new CLI version is available once a day [#1190](https://github.com/keptn/keptn/issues/1190)
+- Check Keptn and Kubernetes compatibility based on K8s cluster version before installing [#1326](https://github.com/keptn/keptn/issues/1326)
+
+</p>
+</details>
+
+<details><summary>Integrations / Keptn contrib</summary>
+<p>
+
+- *dynatrace-SLI-service:*
+  - Ensure compatibility with new metrics v2 api (/api/v2/metrics/query) [#1282](https://github.com/keptn/keptn/issues/1282)
+- *dynatrace-service:*
+  - Automatically create custom alert rules [#1265](https://github.com/keptn/keptn/issues/1265)
+  - Support of new API for Dashboards [#1358](https://github.com/keptn/keptn/issues/1358)
+- *servicenow-service:*
+  - servicenow-service supports Keptn 0.6.x in [0.2.0](https://github.com/keptn-contrib/servicenow-service/releases/tag/0.2.0)
+
+</p>
+</details>
+
+## Fixed Issues
+- Check K8s cluster version only for full installation [#1398](https://github.com/keptn/keptn/issues/1398)
+- Fixed memory leak in *mongodb-datastore* [#1440](https://github.com/keptn/keptn/issues/1440)
+- Use right mount path for persistant volume in *mongodb-datastore* [#1360](https://github.com/keptn/keptn/issues/1360)
+
+
+## Good to know / known limitations
+
+Addressed limitations:
+* ~~Distributors can not automatically reconnect to NATS cluster [#1209](https://github.com/keptn/keptn/issues/1209)~~
+* ~~Dynatrace SLI Service / Metrics API will change at the end of Q1 [#1282](https://github.com/keptn/keptn/issues/1282)~~ 
```

Comment:
we've updated the service, so this is also gone

Reply:
Please take a look at the updated version.